Minor updates to access reviews(Beta) (#12973)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewinstancedecisionitemresource.md

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> Every decision item in an access review represents a principal's access to a resource. accessReviewInstanceDecisionItemResource represents the resource associated with the decision item. accessReviewInstanceDecisionItemResource is an open type which allows for additional properties to be passed in.

Reviewer: ```suggestion
Every decision item in an access review represents a principal's access to a resource that is identified by an accessReviewInstanceDecisionItemResource object. accessReviewInstanceDecisionItemResource is an open type that allows for other properties to be passed in.
```
